mcp-client/simpleOAuthClient.ts

```diff
@@ -24,10 +24,7 @@ if (process.env.http_proxy) {
   setGlobalDispatcher(dispatcher);
 }
 
-/*
- * CURITY EDIT: Updated default server URL
- */
-const DEFAULT_SERVER_URL = 'http://mcp.demo.example';
+const DEFAULT_SERVER_URL = 'http://localhost:3000/mcp';
```

Comment:
Reset to defaults here and use an environment variable in package.json, which seems to be their intent